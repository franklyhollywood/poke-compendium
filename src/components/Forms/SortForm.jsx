import React from 'react';

export default function SortForm() {
	return (
		<div>
			<select aria-label='sort' value={sortPokemon}></select>
		</div>
	);
}
