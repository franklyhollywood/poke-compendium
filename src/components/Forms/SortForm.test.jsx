import { screen, render } from '@testing-library/react';
import SortForm from './SortForm.jsx';

it('testing my filter', () => {
	render(<SortForm />);
	const dropdown = screen.getAllByLabelText('sort');
	expect(dropdown).toMatchSnapshot();
});
