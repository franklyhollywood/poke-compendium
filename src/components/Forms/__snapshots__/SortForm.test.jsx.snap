// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing my filter 1`] = `
Array [
  <select
    aria-label="sort"
  >
    <option
      value="asc"
    >
      ASC
    </option>
    <option
      value="desc"
    >
      DESC
    </option>
  </select>,
]
`;
